Makefile

```diff
@@ -28,18 +32,23 @@ package:
 	python setup.py sdist bdist_wheel
 	twine check dist/*
 
+install_dev:
+	rm -rf $(VENV)
+	python -m venv $(VENV)
+	$(PYTHON) -m pip install --upgrade pip
+	$(PYTHON) -m pip install -e ".[dev, test]"
+
+install_test_pandas:
+	pip install -e ".[test, notebook]"
```

Comment:
Since Pandas is always present, it makes more sense to just name it `install_test`